### ISSUE TITLE
[$500] Workflows - Offline indicator is not at the bottom of the page

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#37721]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Found when validating PR :** https://github.com/Expensify/App/pull/37391

**Version Number:** 1.4.47-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team

## Action Performed:
Precondition:
- User is the owner of Collect workspace.

1. Go to staging.new.expensify.com
2. Go to workspace settings > Collect workspace.
3. Go to Workflows.
4. Go offline.

## Expected Result:
The offline indicator locates at the bottom of the page.

## Actual Result:
The offline indicator locates below the workflow card.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

![Bug6402295_1709609668564!Screenshot_2024-03-05_at_11 16 19](https://github.com/Expensify/App/assets/115492554/0104c9d7-5e89-4564-bc9a-83871b4d3239)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f79caf6c04f732a8</li>
        <li>Upwork Job ID: 1765150505634082816</li>
        <li>Last Price Increase: 2024-04-25</li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
@johncschuster I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think that this bug might be related to #wave8-collect-admins.
CC @zanyrenney

**Comment by rushatgabhane** [comment]
This is not a bug. It's expected result. cc: @luacmartins

**Comment by luacmartins** [comment]
@Expensify/design is the offline indicator supposed to be where it is in the image above?

**Comment by dannymcclain** [comment]
I would expect it to be at the bottom like on similar screens:

![CleanShot 2024-03-05 at 14 54 52@2x](https://github.com/Expensify/App/assets/7516624/e93f1e8c-36c8-4263-98e8-b10e7f5a92d9)

**Comment by dubielzyk-expensify** [comment]
Yep that looks good to me too. Fixed to the bottom of the "current" view.

**Comment by luacmartins** [comment]
Thanks for confirming! Adding external label

**Comment by shawnborton** [comment]
Yup, I agree with the designers.

**Comment by eh2077** [comment]
> Dupe of #37403

@mkhutornyi Yeah, they very likely share the same root cause. Both of them have solid discussions - this one is about the design part, and the other one covers the coding part. 

@luacmartins  I saw you commented on both. Should we put this one on hold for the other one #37403 created earlier?

**Comment by luacmartins** [comment]
Sure, let's put it on hold in favor of https://github.com/Expensify/App/issues/37403

**Comment by johncschuster** [comment]
This is on hold. Downgrading to `Weekly` while we wait on the above linked issue.

**Comment by johncschuster** [comment]
Same

**Comment by johncschuster** [comment]
Looks like https://github.com/Expensify/App/issues/37403 is now closed. Taking this off hold!

**Comment by eh2077** [comment]
<img width="1440" alt="image" src="https://github.com/Expensify/App/assets/117511920/c74ce224-5e45-47fb-845c-959b11671883">


@johncschuster I think we can close it as it's been fixed by https://github.com/Expensify/App/issues/37403

**Comment by eh2077** [comment]
@johncschuster Friendly bump! I think it's good to close this issue.

**Comment by luacmartins** [comment]
Nice! Thanks for testing again!
